fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -2097,6 +2097,17 @@ opt_default_value ::=
     {:
         RESULT = ColumnDef.DefaultValue.NULL_DEFAULT_VALUE;
     :}
+    | KW_DEFAULT function_name:fn_name LPAREN RPAREN
```

Comment:
Why not use function call rule directly?

Reply:
FunctionCallExpr needs to pass parameter.It is not abstracted as a rule.

Reply:
It seems that MySQL support function with parameter as default values.
```
b BINARY(16)  DEFAULT (UUID_TO_BIN(UUID())),
```

Reply:
https://dev.mysql.com/doc/refman/8.0/en/data-type-defaults.html

Reply:
Yes. in this PR we only support CURRENT_TIMESTAMP/now()  compatible with MySQL. FE implement.
other function syntax will be implement later. This only be could do.

Reply:
In my opinion, only CURRENT_TIMESTAMP can be supported. now() can be supported later with other functions.

Reply:
ok, it will makes code more easy.